packages/zwave-js/src/lib/commandclass/UserCodeCC.ts

```diff
@@ -271,29 +313,8 @@ function persistUserCode(
 		valueDB.setMetadata(statusValueId, undefined);
 		valueDB.setMetadata(codeValueId, undefined);
 	} else {
-		// Always create metadata if it does not exist
-		if (!valueDB.hasMetadata(statusValueId)) {
-			valueDB.setMetadata(statusValueId, {
-				...ValueMetadata.Number,
-				label: `User ID status (${userId})`,
-				states: enumValuesToMetadataStates(
-					UserIDStatus,
-					supportedUserIDStatuses,
-				),
-			});
-		}
-		const codeMetadata: ValueMetadata = {
-			...(typeof userCode === "string"
-				? ValueMetadata.String
-				: ValueMetadata.Buffer),
-			minLength: 4,
-			maxLength: 10,
-			label: `User Code (${userId})`,
-		};
-		if (valueDB.getMetadata(codeValueId)?.type !== codeMetadata.type) {
-			valueDB.setMetadata(codeValueId, codeMetadata);
-		}
-
+		// Always create metadata in case it does not exist
```

Comment:
Can't comment on the correct line, but `const supportedUserIDStatuses` isn't used in this method any longer and can be removed.